tests/zenko_e2e/backbeat/ReplicationUtility.js

```diff
@@ -561,6 +563,48 @@ class ReplicationUtility {
             return cb();
         });
     }
+
+    checkReplicationStatus(locationName, cb) {
```

Comment:
Maybe `check` methods should be renamed to something like `getReplicationStatus` since no assertions are happening in the methods. Alternatively, I think it would be good to add the assertions in the helper methods—that way it condenses the test cases. 😸